validator/src/main/java/com/google/daq/mqtt/registrar/LocalDevice.java

```diff
@@ -309,7 +310,7 @@ private Metadata readMetadataWithValidation(boolean validate) {
     final JsonNode instance;
     try (InputStream targetStream = new FileInputStream(metadataFile)) {
       instance = OBJECT_MAPPER.readTree(targetStream);
-      baseVersion = instance.get("version");
+      baseVersion = instance.get(VERSION_PROPERTY_KEY);
       new MessageUpgrader("metadata", instance).upgrade();
```

Comment:
Should this be MessageUpgrader.METADATA_SCHEMA

Reply:
Done.